content/docs/authorization/upgrade.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Upgrade
+linktitle: Upgrade 
+weight: 3
+description: >
+  Upgrade Dell EMC Container Storage Modules (CSM) for Authorization
+---
+
+This section outlines the upgrade steps for Container Storage Modules (CSM) for Authorization.  The upgrade of CSM for Authorization is handled in 2 parts:
+- Upgrading the CSM for Authorization proxy host
```

Comment:
"Upgrading the CSM for Authorization proxy server"

Reply:
fixed